### DISCUSSION TITLE
How do you enable captcha support for a website, but disable it when signing into a native app?

### DISCUSSION THREAD

**Discussion by Jordan-Ellis** [supabase/supabase#21631] [Questions]
I'm currently developing a website and a native companion app using Supabase, both with shared login functionality. I'd like to incorporate captcha support for the website login, but not for the native app. Is there a way to selectively enable captcha only for the website? Any guidance or solutions would be appreciated!

**Comment by silentworks** [comment]
I don't think this is possible at the moment if both are making use of the same Supabase project.

**Reply by igorlanko** [reply]
@silentworks, could you recommend a workaround for this? I'm looking for ideas for the issue (https://github.com/orgs/supabase/discussions/27181) where I can't log into the client app (Electron desktop) because Turnstile does not work from app URL schemas (only expects `https://` as an origin).

**Reply by jack-hanlon** [reply]
I am having the same issue. Before it was working for me because I believe capacitor was running a http://localhost server for the webview, but now I think they changed it so my users cannot sign in to the iOS app

**Reply by igorlanko** [reply]
I couldn't figure this out, so I ended up going down the web auth + deep back linking to the app route.

**Reply by jack-hanlon** [reply]
I put a bug report in the capacitor forum, but I can't do that because my app is offline first. I am hoping supabase allows a way to bypass captcha for a specific hostname, that would solve the problem for me.
